ui/src/app.tsx

```diff
@@ -82,6 +83,7 @@ export const App = () => (
           <Route path="collections/:id" element={<CollectionDetails />} />
           <Route path="*" element={<UnderConstruction />} />
         </Route>
+        <Route path="/:slug" element={<InfoPageContainer />} />
```

Comment:
Checkout this generic route, BE can return page details for any slug here and we will try render it.

Reply:
This is great, thanks!